cg/meta/archive/archive.py

```diff
@@ -305,6 +305,9 @@ def retrieve_spring_files_for_case(self, case_id: str) -> None:
             except MissingFilesError as error:
                 LOG.warning(str(error))
                 continue
+            except SampleFilesCurrentlyArchivingError as error:
+                LOG.warning(f"Sample {sample.internal_id} will not be retrieved: {error}")
+                continue
```

Comment:
I am not sure if this is the desired behaviour. What do you think @Vince-janv @islean

Reply:
The behaviour being skip samples that are still archiving? I think that makes sense. But this logging looks quite verbose. Wille we not pring both line 309 and 324?